pyproject.toml

```diff
@@ -33,6 +36,9 @@ hdf5storage = "^0.1.18"
 psutil = "^5.9.0"
```

Comment:
Seems like `poetry update` bumped `pyrevolve` to version `2.2.2 , but `2.2.2` is broken (doesn't install on my Mac): https://github.com/devitocodes/pyrevolve/releases. Looks like `2.2.3` fixes the issue, so it could help to pin `pyrevolve>=2.2.3`. That might be causing the CI error?

For reference, `pyrevolve` dependency comes from `devito`.

```
devito 4.7.1 Finite Difference DSL for symbolic computation.
└── pyrevolve >=2.1.3
    └── contexttimer *
```

Reply:
Actually, this is an issue on the `main` branch, so I will just make a separate PR.